_includes/release-assets.html

```diff
@@ -31,13 +31,31 @@
 </td>
 <td>
 	<ul class="row">
+		{% assign signed_firefox = include.assets | where_exp:"item", "item.name contains '-firefox.'" | first %}
+		{% assign unsigned_firefox = include.assets | where_exp:"item", "item.name contains '-firefox-unsigned'" | first %}
 
-		{% assign firefox = include.assets | where_exp:"item", "item.name contains '-firefox.'" | first %}
-		{% if firefox != nil %}
-			<li><a href="{{ firefox.browser_download_url}}" target="_blank">Firefox</a></li>
+		{% if include.latest %}
+		
+		{% if signed_firefox != nil %}
+			<li><a href="{{ signed_firefox.browser_download_url}}" target="_blank">Firefox</a></li>
 		{% else %}
 			<li style="text-decoration: line-through">Firefox</li>
 		{% endif %}
+		{% if unsigned_firefox != nil and unsigned_firefox.name > signed_firefox.name %}
```

Comment:
Also, I'm not sure it that's possible, but what if `unsigned_firefox != nil` and `signed_firefox == nil`? Then I assume you'll get an error due to `nil`-deref.